exchange/gdpr.go

```diff
@@ -0,0 +1,92 @@
+package exchange
+
+import (
+	"encoding/json"
+	"strings"
+
+	"github.com/mxmCherry/openrtb"
+)
+
+// ExtractGDPR will pull the gdpr info from an openrtb request
+func extractGDPR(bidRequest *openrtb.BidRequest, usersyncIfAmbiguous bool) (gdpr int, consent string, err error) {
+	var re regsExt
+	if bidRequest.Regs != nil {
+		err = json.Unmarshal(bidRequest.Regs.Ext, &re)
+	}
+	if re.GDPR == nil || err != nil {
+		if usersyncIfAmbiguous {
```

Comment:
I think this is backwards... if `usersyncIfAmbiguous` is true, shouldn't the "effective" `gdpr` be `0`?

Reply:
I even had a test for this, but my test had the same bug. :)